src/System.Security.Cryptography.Encoding/src/Resources/Strings.resx

```diff
@@ -67,6 +67,93 @@
+  </data>
+  <data name="Cryptography_AsnSerializer_MultipleAsnTypeAttributes" xml:space="preserve">
+    <value>Field '{0}' on type '{1}' has multiple attributes deriving from '{2}' when at most one is permitted.</value>
+  </data>
+  <data name="Cryptography_AsnSerializer_NoJaggedArrays" xml:space="preserve">
+    <value>Type '{0}' cannot be serialized or deserialized because it is an array of arrays.</value>
+  </data>
+  <data name="Cryptography_AsnSerializer_NoMultiDimensionalArrays" xml:space="preserve">
+    <value>Type '{0}' cannot be serialized or deserialized because it is a multi-dimensional array.</value>
+  </data>
+  <data name="Cryptography_AsnSerializer_NoOpenTypes" xml:space="preserve">
+    <value>Type '{0}' cannot be serialized or deserialized because it is not sealed or has unbound generic parameters.</value>
+  </data>
+  <data name="Cryptography_AsnSerializer_Optional_NonNullableField" xml:space="preserve">
+    <value>Field '{0}' on type '{1}' is declared [OptionalValue], but it can not be assigned a null value.</value>
+  </data>
+  <data name="Cryptography_AsnSerializer_PopulateFriendlyNameOnString" xml:space="preserve">
+    <value>Field '{0}' on type '{1}' has [ObjectIdentifier].PopulateFriendlyName set to true, which is not applicable to a string.  Change the field to '{2}' or set PopulateFriendlyName to false.</value>
+  </data>
+  <data name="Cryptography_AsnSerializer_SetValueException" xml:space="preserve">
+    <value>Unable to set field {0} on type {1}.</value>
+  </data>
+  <data name="Cryptography_AsnSerializer_SpecificTagChoice" xml:space="preserve">
+    <value>Field '{0}' on type '{1}' has specified an implicit tag value via [ExpectedTag] for [Choice] type '{2}'. ExplicitTag must be true, or the [ExpectedTag] attribute removed.</value>
+  </data>
+  <data name="Cryptography_AsnSerializer_UnexpectedTypeForAttribute" xml:space="preserve">
+    <value>Field '{0}' of type '{1}' has an effective type of '{2}' when one of ({3}) was expected.</value>
+  </data>
+  <data name="Cryptography_AsnSerializer_UtcTimeTwoDigitYearMaxTooSmall" xml:space="preserve">
+    <value>Field '{0}' on type '{1}' has a [UtcTime] TwoDigitYearMax value ({2}) smaller than the minimum (99).</value>
+  </data>
+  <data name="Cryptography_AsnSerializer_UnhandledType" xml:space="preserve">
+    <value>Could not determine how to serialize or deserialize type '{0}'.</value>
+  </data>
+  <data name="Cryptography_AsnWriter_EncodeUnbalancedStack" xml:space="preserve">
+    <value>Encode cannot be called while a Sequence or SetOf is still open.</value>
+  </data>
+  <data name="Cryptography_AsnWriter_PopWrongTag" xml:space="preserve">
+    <value>Cannot pop the requested tag as it is not currently in progress.</value>
+  </data>
+  <data name="Cryptography_Der_Invalid_Encoding" xml:space="preserve">
+    <value>ASN1 corrupted data.</value>
+  </data>
+  <data name="Cryptography_Invalid_IA5String" xml:space="preserve">
```

Comment:
Instead of introducing both the generator and the serializer in this PR, can we go ahead and drop the serializer?